src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
@gabrielschulhof i guess code worth i thousands words - https://github.com/jquery/jquery/pull/2335, https://github.com/jquery/jquery/pull/2336

@scottgonzalez 
1. We wouldn't need to change docs
2. We should be back-compat if possible
3. We shouldn't add needless logic & bytes
4. User could explore new properties of the original event when they comes up, without for us to do anything or change our stub
5. There is already possible use-cases, users don't use them because they can't which doens't mean they aren't useful - http://jsfiddle.net/rs5jn1bq/
6. Consistency, this should be important and it correlates with basic concepts of the DOM events described in the spec